script.js

```diff
@@ -979,11 +983,15 @@ require([
 
   // when township changes, reset the other dropdowns.
   on(townshipSelect, "change", function (evt) {
+    resetElements(townshipSelect, false);
     var type = evt.target.value;
     var i;
     for (i = rangeSelect.options.length - 1; i >= 0; i--) {
       rangeSelect.remove(i);
     }
+    for (j = sectionSelect.options.length - 1; j >= 0; j--) {
+      sectionSelect.remove(j);
+    }
```

Comment:
I honestly think that we should just forget about the section: hard code 1-36 and then forget about it.